assignment-6/src/App.js

```diff
@@ -0,0 +1,77 @@
+import { useEffect, useState } from 'react';
+import './styles.css';
+import { CardComponent, TitleComponent, SearchBar } from './Components';
+
+const CardContainer = ({ userData }) => {
+  return (
+    <div className='container'>
+      <div className='row'>
+        {
+          userData.map((singleUser) => {
+            // console.log(singleUser);
+            return <CardComponent singleUser={singleUser} key={singleUser.id}/>
+          })
+        }
+      </div>
+    </div>
+  )
+}
+
+const usernames = [
+  "akshaymarch7", 
+  "ap221882", 
+  "nikitaj-57", 
+  "Pujarini", 
+  "aditifarkya",
+  "gavandivya",
+  "soumyagangamwar",
+  "Bhallora"
+]
+
+const BodyComponent = () => {
+  // UserData will store the data fetched using Github API
+  const [ userData, setUserData ] = useState([]);
+
+  const fetchUserDetails = async() => {
+    let response = await Promise.all(
+      usernames.map(async(name) => {
+        const userInfo = await fetch(`https://api.github.com/users/${name}`);
+        const data = await userInfo.json();
+        return data;
+        // console.log(data);
+      })
+    )
+    setUserData(response);
+    // console.log(response);
+  }
+
+  useEffect(() => {
+    fetchUserDetails();
+  },[])
+  
+  return (
+    <div>
+      <nav class="navbar navbar-dark bg-dark py-0" >
+        <div class="container-fluid">
+            <TitleComponent/>
+            <SearchBar userData={userData} setUserData={setUserData}/>
+        </div>
+      </nav>
+      <CardContainer userData={userData}/>
```

Comment:
Can you test the scenario where, after searching, if you delete the search text, all of the users' cards reappear?

Reply:
And also please test the scenario where type aditi and hit search button, now remove aditi text and without hitting search button change the search text to divya. Now hit the search button. Does the search result work. ?